homeassistant/components/intent/timers.py

```diff
@@ -207,6 +219,8 @@ def start_timer(
         seconds: int | None,
         language: str,
         name: str | None = None,
+        conversation_command: str | None = None,
+        conversation_agent_id: str | None = None,
```

Comment:
Let's validate that we have an agent ID when a command is passed in.

Reply:
`None` could get passed in for the default agent though :/